src/assets/css/signup.scss

```diff
@@ -0,0 +1,45 @@
+.signuppage{
+    border: 0px solid black;
+    padding: 5%;
+}
+
+.form-contents{
+    border: 1px solid #00bfff;
```

Comment:
Please kindly format this file with indentation of 2 spaces and same for every other .scss file

Reply:
let me work on this @NicholusMuwonge

Reply:
@NicholusMuwonge this has been resolved